### PR TITLE
Merge create and update into a single function + delete old jobs duri…

### PR DESCRIPTION
**Pull request by jdartigalongue**
…ng update + cleanup after upgrades.

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/911#" title="Author self-approved">jdartigalongue</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [jdartigalongue]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_create.go [hunk 1]
```diff
@@ -582,82 +577,19 @@ var createPolarisCmd = &cobra.Command{
...
-		var deployments []deploy
-
-		dbComponents, err := polaris.GetPolarisDBComponents(baseURL, *polarisObj)
-		if err != nil {
-			return err
-		}
-		deployments = append(deployments, deploy{name: "Polaris DB", obj: dbComponents})
-
-		polarisComponents, err := polaris.GetPolarisComponents(baseURL, *polarisObj)
+		oldPolaris, err := getPolarisFromSecret()
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_create.go]: ensure polaris already called this method..

File: pkg/synopsysctl/utils.go [hunk 2]
```diff
@@ -357,3 +361,119 @@ func printWaitingDots(delay time.Duration, stop <-chan struct{}) {
 		}
 	}
 }
+
+func cleanupByLabel(namespace, labelSelector string) error {
```
Comment (msenmurugan) [inline pkg/synopsysctl/utils.go]: delete all run time objects + secrets + pvc's won't do the same?

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/911#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/911#pullrequestreview-294407954" title="LGTM">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [jdartigalongue,msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**